packages/amplify-graphql-model-transformer/src/resources/amplify-dynamodb-table/amplify-table-manager-lambda/amplify-table-manager-handler.ts

```diff
@@ -130,6 +130,27 @@ const processOnEvent = async (event: AWSCDKAsyncCustomResource.OnEventRequest):
   let result;
   switch (event.RequestType) {
     case 'Create':
+      if (tableDef.isImported) {
+        // TODO: Add import validation
+        console.log('Initiating table import process');
+        console.log('Fetching current table state');
+        console.log(`Table name: ${tableDef.tableName}`);
+        const describeTableResult = await ddbClient.describeTable({ TableName: tableDef.tableName });
+        if (!describeTableResult.Table) {
+          throw new Error(`Could not find ${tableDef.tableName} to update`);
+        }
+        log('Current table state: ', describeTableResult);
```

Comment:
Any sensitive data to worry about in the `describeTableResult`? I'm assuming not, but let's be careful about what we log.